### PR TITLE
Remove default values from the install prow script

### PR DESCRIPTION
**Pull request by strekm**
**Description**

Changes proposed in this pull request:

- Remove default values for BUCKET_NAME, KEYRING_NAME, and ENCRYPTION_KEY_NAME from the install-prow.sh script
- Update docu

**Related issue(s)**
See #68

### CODE REVIEW

File: prow-configurations/README.md [hunk 1]
```diff
@@ -29,7 +29,17 @@ To generate a new token, go to the **Settings** tab of a given GitHub account an
...
 
-2. Run the following script to start the installation process:
+2. Export the variables, where:
+ - **BUCKET_NAME** is a GCS bucket in the Google Cloud project that is used to store Prow Secrets.
+ - **KEYRING_NAME** is the KMS key ring.
+ - **ENCRYPTION_KEY_NAME** is the key name in the key ring that is used for data encryption.
+
+ >**NOTE:** For more details on setting up Google Cloud Project refer to [Prow Secrets management](https://github.com/kyma-project/test-infra/blob/master/docs/prow-secrets-management.md)
+
+ The Prow installation script assumes the following service account keys, encrypted with the **ENCRYPTION_KEY_NAME** from **KEYRING_NAME**, is present in **BUCKET_NAME**
```
Comment (jakkab) [inline prow-configurations/README.md]:
```suggestion
 The Prow installation script assumes that the following service accounts exist:
 -  **sa-gke-kyma-integration** with a role allowing to create Kubernetes clusters
 - **sa-vm-kyma-integration with roles allowing to provision Virtual Machines
with their keys, encrypted using the **ENCRYPTION_KEY_NAME** from **KEYRING_NAME**, being present in **BUCKET_NAME**
```

File: prow-configurations/README.md [hunk 2]
```diff
@@ -29,7 +29,17 @@ To generate a new token, go to the **Settings** tab of a given GitHub account an
...
+ - **sa-gke-kyma-integration** with a role allowing to create kubernetes clusters.
```
Comment (jakkab) [inline prow-configurations/README.md]: ~~- **sa-gke-kyma-integration** with a role allowing to create kubernetes clusters.~~

Comment (jakkab) [inline prow-configurations/README.md]:
```suggestion
```

File: development/README.md [hunk 3]
```diff
@@ -29,7 +29,19 @@ To generate a new token, go to the **Settings** tab of a given GitHub account an
 In the new window, select all scopes and click **Generate token**.
 >**NOTE:** Create a separate bot account instead of using your personal one. If the Prow bot account is the same as the account that creates a job-triggering comment, the job is not triggered.
 
-2. Run the following script to start the installation process:
+2. Export the variables, where:
```
Comment (tomekpapiernik) [inline development/README.md]:
```suggestion
2. Export the environment variables:
```

File: prow/README.md [hunk 4]
```diff
@@ -32,7 +32,18 @@ The obligatory Secrets include:
 gcloud container clusters get-credentials {CLUSTER_NAME} --zone={ZONE_NAME} --project={PROJECT_NAME}
 ```
 
-2. Run the following script to start the installation process:
+2. Export the variables, where:
```
Comment (tomekpapiernik) [inline prow/README.md]:
```suggestion
2. Export the environment variables:
```

File: development/README.md [hunk 5]
```diff
@@ -98,7 +98,7 @@ prow
 ...
 ```
 
-### Convention of naming jobs
+### Convention for naming jobs
```
Comment (klaudiagrz) [inline development/README.md]: Thank you for correcting that, it was my bad 😉

**Pull request merged**